server/app/services/rpc/node_handler.rb

```diff
@@ -0,0 +1,45 @@
+require_relative 'fixnum_helper'
+
+module Rpc
+  class NodeHandler
+    include Celluloid
+    include FixnumHelper
+
+    def initialize(grid)
+      @grid = grid
+    end
+
+    def get(id)
+      node = @grid.host_nodes.find_by(node_id: id)
+      if node
+        template = Tilt.new('app/views/v1/host_nodes/_host_node.json.jbuilder')
+        JSON.parse(template.render(nil, node: node))
```

Comment:
Yes and no. This PR does not include enhanced serializers... but the follow up PR does.